### PR TITLE
OneWay bug fix

### PR DESCRIPTION
**Pull request by AlexSananka**
@dannyparsons  I have made the minor edits here,You can merge this.

### CODE REVIEW

**Comment by dannyparsons** [comment]
there are no changes here?

**Comment by AlexSananka** [comment]
I made the changes on my master so I just merged my master to this branch

**Comment by AlexSananka** [comment]
@stevenndungu I made the suggested changes,please test this,I think it is ready.

**Comment by stevenndungu** [comment]
Great, let me test it

File: instat/sdgOneWayFrequencies.Designer.vb [hunk 1]
```diff
@@ -407,7 +398,6 @@ Partial Class sdgOneWayFrequencies
     Friend WithEvents rdoLine As RadioButton
     Friend WithEvents rdoDot As RadioButton
     Friend WithEvents rdoBar As RadioButton
-    Friend WithEvents ucrSaveGraph As ucrSave
     Friend WithEvents lblTitle As Label
     Friend WithEvents lblHjust As Label
     Friend WithEvents lblVjust As Label
```
Comment (lilyclements) [inline instat/sdgOneWayFrequencies.Designer.vb]: I don't feel it's efficient to make a pull request just for this one tiny thing - so can you reduce the sdg dialog size to (381, 362)?

File: instat/dlgOneWayFrequencies.vb [hunk 2]
```diff
@@ -71,8 +71,8 @@ Public Class dlgOneWayFrequencies
         'setting rdoBoth 
         ucrPnlFrequencies.AddFunctionNamesCondition(rdoBoth, "sjplot")
         ucrPnlFrequencies.AddFunctionNamesCondition(rdoBoth, "sjtab")
-        ucrPnlFrequencies.AddToLinkedControls(ucrChkFlip, {rdoGraph, rdoBoth}, bNewLinkedDisabledIfParameterMissing:=True, bNewLinkedAddRemoveParameter:=True)
-        ucrPnlFrequencies.AddToLinkedControls(ucrSaveGraph, {rdoGraph, rdoBoth}, bNewLinkedDisabledIfParameterMissing:=True, bNewLinkedAddRemoveParameter:=True)
+        ucrPnlFrequencies.AddToLinkedControls(ucrChkFlip, {rdoGraph, rdoBoth}, bNewLinkedHideIfParameterMissing:=True)
+        ucrPnlFrequencies.AddToLinkedControls(ucrSaveGraph, {rdoGraph, rdoBoth}, bNewLinkedHideIfParameterMissing:=True)
```
Comment (lilyclements) [inline instat/dlgOneWayFrequencies.vb]: Out of interest, why isn't the parameter being added/removed anymore?

**Comment by stevenndungu** [comment]
Otherwise it quite good.

**Comment by AlexSananka** [comment]
@stevenndungu That's a good idea,I am going to change the prefix.

I am updating the twoway on the threevariable branch.

File: instat/sdgOneWayFrequencies.Designer.vb (hunk 1 above)
Comment (lilyclements) [inline instat/sdgOneWayFrequencies.Designer.vb]: I meant, I don't feel it's efficient for me to make a pull request to then merge it to this branch just for this small thing.
It'll take me about ten minutes just to make this one minor edit, however, and I'm meant to be working on non-R-Instat work. Please can you make this change. If it were more changes than this small one then I would do a pull request.

File: instat/sdgOneWayFrequencies.Designer.vb [hunk 3]
```diff
@@ -355,7 +355,7 @@ Partial Class sdgOneWayFrequencies
         '
         Me.AutoScaleDimensions = New System.Drawing.SizeF(6.0!, 13.0!)
         Me.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font
-        Me.ClientSize = New System.Drawing.Size(365, 350)
+        Me.ClientSize = New System.Drawing.Size(365, 323)
```
Comment (lilyclements) [inline instat/sdgOneWayFrequencies.Designer.vb]: This is now too small and cuts off the ucrBase! Please change it to the size I specified earlier

File: instat/sdgOneWayFrequencies.Designer.vb (hunk 1 above)
Comment (AlexSananka) [inline instat/sdgOneWayFrequencies.Designer.vb]: Sure

**Comment by lilyclements** [comment]
@dannyparsons This is ready for you to review

File: instat/dlgOneWayFrequencies.vb (hunk 2 above)
Comment (dannyparsons) [inline instat/dlgOneWayFrequencies.vb]: I think because the check box doesn't control a parameter - just the names of different parameters.

**Pull request merged**